app/javascript/stylesheets/components/Post.scss

```diff
@@ -118,4 +118,15 @@
       color: $astuto-black;
     }
   }
+}
+
+@media (prefers-color-scheme: dark) {
+
+  .postDescription {
+    color: white !important;
```

Comment:
Are all these `!important` necessary?

Reply:
I'm not entirely sure why sometimes it's necessary and other times it works without the !important. I've deleted them wherever possible.

Reply:
It's probably because of CSS specificity, I removed some of them with the last commit.